airbyte-webapp/src/components/Modal/Modal.module.scss

```diff
@@ -7,23 +7,39 @@
   }
 }
 
-.modal {
-  animation: fadeIn 0.2s ease-out;
-  position: absolute;
+.backdrop {
```

Comment:
There may be stylelint issues when that PR gets merged.

Reply:
Oh, good to know!  What in this file would cause the issue do you think?

Reply:
I don't recall the linters preference, but it either prefers `: 0` or `: 0px`
It may not be an issue at all!

Reply:
rebased against master and ran linter... seems like `0` was the key :)